demo/button/basic/playground

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-console, no-undef, react/prop-types, react/react-in-jsx-scope, react/jsx-no-undef */
+
+const capitalize = (s) => s[0].toUpperCase() + s.slice(1, s.length)
+
+const layouts = ['full']
+const types = ['primary', 'secondary', 'accent', 'ghost', 'flat']
+const sizes = ['small', 'medium', 'large']
+
+const Icon = (
```

Comment:
Even for demo purposes only, I'd rather use an Icon provided by [SVG-iconset](https://github.com/SUI-Components/svg-iconset) repo so it might help as an implementation example.